addons/src/addons.c

```diff
@@ -652,9 +652,7 @@ static GtkWidget *plugin_addons_configure(G_GNUC_UNUSED GeanyPlugin *plugin, Gtk
 
 	gtk_widget_show_all(vbox);
 
-#if ! GTK_CHECK_VERSION(2, 10, 0)
 	gtk_widget_hide(check_systray);
-#endif
```

Comment:
Looks backwards, should you also remove the body?  I guess the idea was not to add the option when the feature wasn't supported, wasn't it?

Reply:
This is so me...you are completely right.
Thanks for spotting, fixed.

Funny thing, this attempt to hide the option didn't work at all because in Geany itself we call `gtk_widget_show_all()` when creating the plugin preferences dialog and so it was shown again.
Anyway, GTK2 is long time ago.